tests/uvbeam/test_initializers.py

```diff
@@ -40,7 +41,7 @@ def uvb_healpix_kw():
 
 @pytest.fixture()
 def uvb_efield_kw():
-    return {"feed_array": ["x", "y"]}
+    return {"feed_array": ["x", "y"], "feed_angle": [np.pi / 2, 0]}
```

Comment:
Need a test for initializing with a mount_type.

Reply:
Added in 4bd22a9260ec0b4845b31f0bc4d84d7f61e23587.